app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Amusment Park</h1>
+  <%= @amusement_park.name %>
+  <%= @amusement_park.admission_cost %>
+
+<% @amusement_park.rides.each do |ride| %>
+  <%= ride.mechanic_to_ride %>
+<% end %>
```

Comment:
Don't forget the fact that this view is an HTML page and in saying that, every element in an HTML page but be enclosed in HTML tags, currently only your heading is enclosed in HTML.

You want to also add a description to everything on the page so we know what it is. I know what it is because I'm looking at the code, but if I were to view this page in the browser it would be just a bunch of ambiguous text and numbers that had no meaning. Not a very useful page if you ask me.